templates/base.html

```diff
@@ -8,8 +8,12 @@
         <link rel="stylesheet" href="{% static 'bootstrap/css/bootstrap.css' %}" type="text/css"/>
         <link rel="stylesheet" href="{% static 'google-code-prettify/prettify.css' %}" type="text/css"/>
         <link rel="stylesheet" href="{% static 'selectize/dist/css/selectize.bootstrap3.css' %}" data-theme="bootstrap3">
-        <link rel="stylesheet" href="{% static 'own-style/own.css' %}" data-theme="bootstrap3">
         <link rel="stylesheet" href="{% static 'ksp_login/css/base.css' %}" type="text/css" />
+        <link rel="stylesheet" href="{% static 'own-style/own.css' %}">
+        <link rel="stylesheet" href="{% static 'unslider/unslider.css' %}" type="text/css" />
+        <link rel="stylesheet" href="{% static 'unslider/unslider-dots.css' %}" type="text/css" />
+        <link href="//fonts.googleapis.com/css?family=Titillium+Web" rel="stylesheet">
```

Comment:
Dal som zatial Source Sans Light, ktoré vyzerá podľa mňa fakt dobre na väčšine textov...
Ale na nadpisy a do menu to asi chce niečo iné..